Assets/Scripts/с#/SpaceEx5/Space5OnOff.cs

```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
+
+public class Space5OnOff : MonoBehaviour
+{
+    [SerializeField] private GameObject _cube;
+
+    private void Update()
+    {
+        TestCube6();
+    }
+
+    public virtual void TestCube6()
+    {
+        if (Input.GetKeyDown(KeyCode.Space))
+        {
+            if (!_cube.activeSelf)
+                _cube.SetActive(true);
+            else
+                _cube.SetActive(false);
```

Comment:
Here you can get rid of the block if

_cube.SetActive(!_cube.activeSelf);

Reply:
+?

Reply:
it works if the script is disabled (Space5OnOff)